packages/suite/src/support/messages.ts

```diff
@@ -6640,6 +6640,10 @@ export default defineMessages({
         id: 'TR_TXID',
         defaultMessage: 'TX ID',
     },
+    TR_TXID_RBF: {
+        id: 'TR_TXID_RBF',
+        defaultMessage: 'TX ID (rbf)',
```

Comment:
@Hermez-cz What shall be the default string here?

Please not: this will be visible for "Finalize TX" as well (beside RBF)